ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/stun.schema.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  stun:
```

Comment:
Please add descriptions to the schema. This is our documentation, so it should contain some information to guide the user. No need for descriptions if it just repeats the key name, but for anything where there could be some doubt, we should have it.

Reply:
I think you are referring to something like [this](https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/aaa_authentication.schema.yml#L17) for the description to have some CLI? I can do that for client / server seems reasonable.

Reply:
If the descriptions are more or less just repeating the context of the keys and their names, they have limited values. I will ask @JulioPDX to review the schema here and fix / decide if we even need descriptions.

Reply:
Taking a look at this now